tests/unit/cn/cn_tree_cursor_test.c

```diff
@@ -298,14 +297,13 @@ MTF_DEFINE_UTEST_PREPOST(cn_tree_cursor_test, basic, pre_test, post_test)
     struct cn_cursor cur = {
         .cncur_seqno = 10,
     };
-    struct kvs_cparams cp = kvs_cparams_defaults();
     struct cn_tree_node tn;
     struct route_node *rnode;
     const char ekey = 'z';
 
     struct kv kv[2];
 
-    tree.ct_route_map = route_map_create(cp.fanout);
+    tree.ct_route_map = route_map_create(128);
```

Comment:
For the sake of readability, does it make sense to just use FANOUT_MAX here?

Reply:
Probably, I think this got "fixed" long be I settled on using CN_FANOUT_MAX to size the route map :-)

Reply:
Done.